cmd/save_test.go

```diff
@@ -147,7 +147,6 @@ func TestSaveRun(t *testing.T) {
 		{"add rows, save", "me/movies", "testdata/movies/dataset.json", "testdata/movies/body_twenty.csv", "Added 10 more rows", "Adding to the number of rows in dataset", true, false, true, "dataset saved: peer/movies@/map/Qmf516nREprnwE3YnBpZd1y9DkS6e2ktKYDLMtz4L4MgMX\nthis dataset has 1 validation errors\n", "", ""},
 		{"no changes", "me/movies", "testdata/movies/dataset.json", "testdata/movies/body_twenty.csv", "trying to add again", "hopefully this errors", false, false, true, "", "error saving: no changes", ""},
 		{"add viz", "me/movies", "testdata/movies/dataset_with_viz.json", "", "", "", false, false, false, "dataset saved: peer/movies@/map/QmXDKr4ryBuzXFhpKPUAe6GmqEacCjgPkG76NyVLpmPDSt\nthis dataset has 1 validation errors\n", "", ""},
-		{"add transform", "me/movies", "testdata/movies/dataset_with_tf.json", "", "", "", false, false, false, "dataset saved: peer/movies@/map/QmaYNBGZUxJJZVFFf9LkqomSZQ5kLRCaprKtNDwGUJ2bYy\nthis dataset has 1 validation errors\n", "", ""},
```

Comment:
Deleting this because it contains the output from the transform script? I think it could be good to have a verification here that `print` is working. If the worry is testing non-vital messages, like the emoji and progress, maybe have the test body trim off text before "running transform..."

Reply:
I've added an explicit test that confirms print is working, with a test failure that gives more meaningful feedback. I think it justifies dropping this test.

Reply:
Fair, ok, LGTM